sips/sip-x/sip-wallet-protocol.md

```diff
@@ -0,0 +1,84 @@
+
+# Preamble
+
+SIP Number: X
+
+Title: Specification of the Wallet Protcol
+
+Author: Friedger Müffke (mail@friedger.de)
+
+Consideration: Technical
+
+Type: Standard
+
+Status: Draft
+
+Created: 20 January 2022
+
+License: CC0-1.0
+
+Sign-off:
+
+# Abstract
+
+Decentralized application do not handle private keys of users. Instead users have wallets that manage private keys, authentication, message signing etc. This SIP specifies a ways to exchange messages between applications and wallets and how to agree on one.
```

Comment:
Either text is fine with me.  RPC is a form of message exchange, and there will almost certainly be apps that use the message-passing paradigm instead of the RPC paradigm for communication.